### ISSUE TITLE
Install Issue with yajl/json_gem

### ISSUE THREAD

**Issue by bitcalc** [bshaffer/oauth2-server-php-docs#54]
In the last commit, you removed the Gemfile, on purpose? Once the Gemfile is put back into place and after the bundle install/update, I'm getting:
~~~
LoadError: cannot load such file -- yajl/json_gem
~~~
Now I don't know much about Ruby, but I want to contribute to the docs, so know a quick and dirty fix?

**Comment by bshaffer** [comment]
@bitcalc I struggled with a plethora of issues after upgrading to ruby 2.x.  What version of ruby do you have installed?

**Comment by bshaffer** [comment]
The quick-and-dirty fix would be to just use markdown and submit the PR, and I can review the changes.

The not-so-quick-and-dirty would be to upgrade to ruby 2, run `bundle update && bundle install`, and then go to town :)

**Comment by bitcalc** [comment]
@bshaffer `ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]`

I did update/install earlier, after a fair few new gems that needed installing... I'll make the changes and then submit, fix the issue on the side!

**Comment by bshaffer** [comment]
Sorry, so were you able to get it working then?

**Comment by bitcalc** [comment]
No, I have that yajl gem installed, but I don't know ruby and don't know what the /json_gem means or is referring to?

**Comment by bshaffer** [comment]
This is the issue I had as well, and is why I upgraded everything... but removing the Gemfile was in error.

If you have `nanoc 3.7.5` installed, the `nanoc` command should work out of the box.  In the meantime I'll see about getting this working again with a `Gemfile` for fresh checkouts

**Comment by bitcalc** [comment]
I have news, `nanoc` works fine when there is NO `Gemfile`
